hash_practice/exercises.py

```diff
@@ -2,28 +2,104 @@
 def grouped_anagrams(strings):
     """ This method will return an array of arrays.
         Each subarray will have strings which are anagrams of each other
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O(n) (or maybe O(n^2) because of the sorting of each word?)
+        Space Complexity: O(n)

```

Comment:
👍 If you're timing the sorting of each word, you can say O(nm log m) where m is the length of each word